.golangci.yml

```diff
@@ -27,6 +27,7 @@ linters:
   - whitespace
   disable:
   - scopelint
+  - testifylint # To many false positives https://github.com/Antonboom/testifylint/issues/67
```

Comment:
another way to disable this specific set of false positives would be something like:
```
- linters: ["testifylint"]
  text: "require must only be used in the goroutine running the test function"
```

Reply:
oh interesting - is that stable?

Reply:
I'm not sure, that string was probably more recently added since that check wasn't in previous testifylint versions, there might be other ways golangcilint allows ignoring lint errors, will have to look